src/components/atoms/modal/modal.tsx

```diff
@@ -0,0 +1,29 @@
+import { useRef, useState, MouseEvent } from 'react';
+import styles from './modal.module.scss';
+
+export const Modal = (): JSX.Element => {
+  const [showModal, setShowModal] = useState(false);
+  const contentRef = useRef<HTMLDivElement>(null);
+  const closeModal = () => setShowModal(false);
+
+  const clickHandler = (event: MouseEvent<HTMLDivElement>) => {
+    if (contentRef.current && !contentRef.current.contains(event.target as Element)) {
+      closeModal();
+    }
+  };
+
+  return (
+  <>
+    <button className="modal-button" onClick={() => setShowModal(!showModal)}>Modal Link</button>
```

Comment:
I know why this is here, but we probably should make this a prop to start with, and demo how to trigger with a button - unless you think just getting it in for now is valu.... a worthwhile isolated step to deliver ;)